lib/CL/devices/level0/level0-driver.cc

```diff
@@ -2932,6 +2932,7 @@ Level0Device::Level0Device(Level0Driver *Drv, ze_device_handle_t DeviceH,
   if (ClDev->atomic_memory_capabilities & CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES)
     OpenCL30Features.append(" __opencl_c_atomic_scope_all_devices");
 
+#ifndef ENABLE_CONFORMANCE
```

Comment:
What's missing from conformance with CBufs? Could be documented somewhere?

Reply:
I think it can be enabled, nothing should be missing